package.json

```diff
@@ -17,7 +17,7 @@
     "url": "https://github.com/Automattic/sensei/issues"
   },
   "engines": {
-    "npm": "^9.3.1",
+    "npm": "^9.3.1 || ^10",
```

Comment:
Could / should we use the same versions as Gutenberg? Given that we're now developing using the version from `trunk`, it's kind of a pain to have to switch Node / npm versions in order to build Sensei vs. Gutenberg.